docs/modules/migrate/pages/data-migration-tool.adoc

```diff
@@ -0,0 +1,176 @@
+
+[source,shell]
+----
+clc -c source.yaml map --name my-map set key-1 value-1
+----
+
+??? _Any possible errors and ways of fixing them here_ ???
+
+Then, you can run the following script on your terminal to populate 1000 entries to the source cluster. This script is 
+for macOS and Linux.
+
+[source,shell]
+----
+for i in {1..1000}; do clc -c source.yaml map --name my-map set key-$i value-$i --quiet; done && echo OK
+----
+
+This one is for Windows users:
+
+[source,shell]
+----
+for /l %x in (1, 1, 100) do clc -c source.yaml map --name my-map set key-%x value-%x --quiet
+----
+
+NOTE: You can also use xref:getting-started:get-started-binary.adoc#step-3-write-data-to-memory[alternative ways] to create a map and put data.
+
+=== Step 3. Start the Target Cluster
+
+The target cluster must be of version Hazelcast 5.3.2 or newer.
+
+Run the following Docker command to start a one-member cluster.
+
+[source,shell]
+----
+docker run -e HZ_NETWORK_PORT_PORT=5901 -p 127.0.0.1:5901:5901 
+           -e HZ_LICENSEKEY="<license>" <1>
+           -e HZ_CLUSTERNAME=target hazelcast/hazelcast-enterprise:{full-version}-slim <2>
+----
+<1> You can find the license key in the `enterprise-trial-license-key.txt` file included in the DMT download package.
+<2> If you want to use the full distribution, remove the `-slim` postfix. See xref:getting-started:editions.adoc#full-and-slim-distributions[Distributions] to learn about the differences between full and slim distributions.
+
+Alternatively, you can use the migration build which is provided in the DMT download package as the target cluster.
+Run the following command in the folder where you extracted the DMT package.
+
+```shell
+$ HZ_CLUSTERNAME=target HZ_NETWORK_PORT_PORT=5901 ./bin/hz start
+```
+
+The cluster tries to start on port `5901` if it's available. It chooses another port otherwise. You can check the port in the cluster logs shown on the terminal.
+
+The configuration file for the target cluster is `target.yaml`, which can be found in the folder where you extracted the DMT package.
+In case the cluster starts on a different port as you notice within the cluster logs, you must change the `address` field in `target.yaml` accordingly to match the port number.
+
+=== Step 4. Start the Migration Cluster
+
+Run the following command in the folder where you extracted the DMT package.
+
+[source,shell]
+----
+$ HZ_NETWORK_PORT_PORT=5702 HZ_CLUSTERNAME=migration ./bin/hz start
+----
+
+The cluster tries to start on port `5702` if it's available. It chooses another port otherwise. You can check the port in the cluster logs shown on the terminal.
+
+The configuration file for the migration cluster is `migration.yaml`, which can be found in the folder where you extracted the DMT package.
+In case the cluster starts on a different port as you notice within the cluster logs, you must change the `address` field in `migration.yaml` accordingly to match the port number.
```

Comment:
Add: The migration.yaml file is in CLC configuration format and used by DMT. DMT and CLC uses the same configuration format. DMT will use the configuration file to be able to connect to migration cluster

Reply:
@srknzl , I can include what is the relevant file for (to connect to the migration cluster). But I am not sure how knowing the configuration format helps the user in some ways?

Reply:
If the migration cluster uses ssl for example, I don't know. It is better mention this.